iam_openapi_spec.yml

```diff
@@ -3407,6 +3408,8 @@ components:
       type: object
       additionalProperties: false
       properties:
+        issuer:
+          type: string
```

Comment:
Can we make this enum?

Reply:
We'll need to edit this every time an issuer is added in that case. Is that fine?